lib/fixer/configuration.rb

```diff
@@ -50,6 +55,8 @@ def reset!
       self.adapter       = DEFAULT_ADAPTER
       self.endpoint      = ENV['FIXER_ENDPOINT'] || DEFAULT_ENDPOINT
       self.user_agent    = DEFAULT_USER_AGENT
+      self.aws           = nil
+      self.queue         = ENV['FIXER_QUEUE'] || DEFAULT_QUEUE
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.